src/components/BikeTagMap.vue

```
@@ -48,33 +49,30 @@ const getMarkers = computed(() =>
         logo: Pin,
+        size: [markerHeight, markerHeight],
+        anchor: [markerHeight/2, markerHeight]
       }))
     : getAllGames.value
         .filter((game) => !!game.boundary.lat) // add gps location to all games
-        .map((game) => ({ point: game.boundary, logo: getLogoUrl.value('', game.logo) })),
+        .map((game) => ({
+          point: game.boundary,
+          logo: getLogoUrl.value('', game.logo),
+          size: ['auto', markerHeight],
+          anchor: [0, markerHeight]
```

ESLint

[error] Insert `,`

(prettier/prettier)